examples/with-ant-design-mobile/README.md

```diff
@@ -2,6 +2,12 @@
 
 This example features how you use [antd-mobile](https://github.com/ant-design/ant-design-mobile) (Ant Design Mobile FrontEnd Framework) with Next.js.
 
+## Preview
+
+Preview the example live on [StackBlitz](http://stackblitz.com/):
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-ant-design-mobile)
```

Comment:
Note that we should only merge this if it works on StackBlitz, that is we fix the missing peer dependency issue.